### PR TITLE
LOK-2195: Doc: Geo Map: User Documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
## Description
Added documentation for the geo maps page.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2195

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: docs/modules/operation/pages/minions/introduction.adoc [hunk 1]
```diff
@@ -44,23 +57,25 @@ NOTE: We recommend using Linux for production environments.
...
 . Click *Locations* in the left navigation menu.
 . In the *Locations* list, click the vertical ellipsis (*⋮*) next to the location you want to add the Minion to.
 . Click *Edit*.
-. Click *Download Certifcate*.
-. Create a permanent directory for the Minion on the device where you want to run it.
-. Copy the downloaded security certificate to the permanent directory.
-Do not change its name.
-. Copy the password in the *Unlock File Password* field.
-. Open the certificate file and, when prompted, enter the password in the field.
+. Click *Download Certificate*.
```
Comment (mikewrosey) [inline docs/modules/operation/pages/minions/introduction.adoc:67]: On the locations page the button says `Download Bundle`

File: docs/modules/operation/pages/minions/introduction.adoc [hunk 2]
```diff
@@ -44,23 +57,25 @@ NOTE: We recommend using Linux for production environments.
...
+. Create a permanent directory for the Minion on the device where you want to run it Select a permanent location for the download (for example, `/minion/minion-default.zip`).
+.. Future Minion restarts require the certificate.
+. Click *Download Bundle* and copy it to the permanent directory.
+. Copy the password from the *Unlock File Password* field.
```
Comment (mikewrosey) [inline docs/modules/operation/pages/minions/introduction.adoc:71]: The password field and copy btn have been removed. The user won't need to do this anymore.

File: docs/modules/operation/pages/minions/introduction.adoc [hunk 3]
```diff
@@ -44,23 +57,25 @@ NOTE: We recommend using Linux for production environments.
...
+. Unzip and open the download bundle and, when prompted, enter the password in the field.
```
Comment (mikewrosey) [inline docs/modules/operation/pages/minions/introduction.adoc:72]: The user will just have to unzip it into the desired directory and run the docker command.

File: docs/modules/operation/pages/minions/introduction.adoc [hunk 4]
```diff
@@ -44,23 +57,25 @@ NOTE: We recommend using Linux for production environments.
...
 . Open a terminal window.
-. In OpenNMS, copy the Docker command.
+. In the OpenNMS UI, copy the Docker run command (`docker compose up -d`).
 . Paste the command into the terminal window.
 . Edit the `PATH_TO_DOWNLOADED_FILE` with the full path to the certificate file.
 . Run the command.
-. In OpenNMS, click *Save*.
+. In the OpenNMS UI, click *Save*.
```
Comment (mikewrosey) [inline docs/modules/operation/pages/minions/introduction.adoc:78]: Which `save` button is this? The minion should just show up after refreshing. Probably we can delete this line.

File: docs/modules/operation/pages/minions/introduction.adoc (hunk 2 above)
Comment (Bonrob2) [reply docs/modules/operation/pages/minions/introduction.adoc:71]: Thanks. They were there earlier today when I was looking at this. Bizarre.

**Pull request closed without merging**